### DISCUSSION TITLE
coApp for Win7 installation triggers antivirus protection

### DISCUSSION THREAD

**Discussion by AdalbertSN** [aclap-dev/video-downloadhelper#789] [Help and Q&A]
### Your question

Hi, I received a report from the antivirus for a "Trojan-PSW.Win64.Disco.evv" contained in the companion App for Windows 7. Here is the installer link:
https://www.virustotal.com/gui/file/185653cb64c0f525ff8f5f6ca3491d7dbd4ffd82a68b0d4f89d6245319a6e92d
and here is the extracted .exe in c:\program files:
https://www.virustotal.com/gui/file/f74bb9881d6ad21cc1d6af94df14eec847f684709144a33369da545cf61b12e6
To be safe, I proceeded to disable the Firefox extension and uninstall the vdh coApp. I'm sorry I can't use it, because I think it's one of the best download addons available...
I may reinstall it in the future if there will be an updated version that passes VirusTotal checks. Bye

### Addon & CoApp version and details

version 9.0.2.3

**Comment by Mattmjs** [comment]
The app is safe to use, it is a false positive from the anti virus. Just white list or allow it to run, whatever the procedure is in
the anti virus.
